edx/edx-app-ios
# Remove the Org / Course number completely on course cards

### Description

[LEARNER-3742](https://openedx.atlassian.net/browse/LEARNER-3742)

### How to test this PR

1. Check Org / Course number should not be visible on course cards
2. Verify Date Format on all screens
